docs/quickstart.rst

```diff
@@ -133,59 +217,61 @@ states it has assigned them:
 
 .. code-block:: text
 
+   # on the monitor virtual machine
+
+   sudo su - postgres
+   export PATH="/usr/lib/postgresql/11/bin/:$PATH"
+
    pg_autoctl show state --pgdata ./monitor
-        Name |   Port | Group |  Node |     Current State |    Assigned State
-   ----------+--------+-------+-------+-------------------+------------------
-   127.0.0.1 |   6010 |     0 |     1 |           primary |           primary
-   127.0.0.1 |   6011 |     0 |     2 |         secondary |         secondary
+       Name |   Port | Group |  Node |     Current State |    Assigned State
+   ---------+--------+-------+-------+-------------------+------------------
+   10.0.1.5 |   5432 |     0 |     1 |           primary |           primary
+   10.0.1.6 |   5432 |     0 |     2 |         secondary |         secondary
 
 This looks good. We can add data to the primary, and watch it get
 reflected in the secondary.
 
 .. code-block:: bash
 
+   # on your local machine
+
    # add data to primary
-   psql -p 6010 \
+   psql TODO -p 6010 \
```

Comment:
Use the following command to get the connection string to use in the application:

```
$ pg_autoctl show uri --formation default
postgres://localhost:4002,localhost:4003,localhost:4001,localhost:4004/postgres?target_session_attrs=read-write
```

In a future version of the tooling you even have:

```
$ pg_autoctl show uri 
      Type |    Name | Connection String
-----------+---------+-------------------------------
   monitor | monitor | port=4000 dbname=pg_auto_failover host=/tmp user=autoctl_node
 formation | default | postgres://localhost:4003,localhost:4002,localhost:4001,localhost:4004/postgres?target_session_attrs=read-write
```

Reply:
In that future version you can even have it in JSON. I am currently pondering a 1.0.6 release with all those improvements and bug fixes as an interim release before the next big feature release with multiple standby support.

```
$ pg_autoctl show uri --json
[
    {
        "uri": "port=4000 dbname=pg_auto_failover host=/tmp user=autoctl_node",
        "name": "monitor",
        "type": "monitor"
    },
    {
        "uri": "postgres://localhost:4003,localhost:4002,localhost:4001,localhost:4004/postgres?target_session_attrs=read-write",
        "name": "default",
        "type": "formation"
    }
]
```

I think it's important you have that information now because we will have to refresh the tutorial with some bits from the new version when we have it.